extension/app/highlighter.js

```diff
@@ -1,36 +1,48 @@
 /* jshint esversion: 6 */
 
-define(['../var/document', '../var/languages'], function(document, languages) {
+define(['../var/document', '../var/languages'], (document, languages) => {
     'use strict';
 
     const INTERVAL = 50; // Interval in milliseconds.
 
+    return {
+        init() {
+            insertStyles();
+            waitForRender('.diff-container').then(() => {
+                prepareLanguageClasses();
+                addCodeTagToElements('.source');
```

Comment:
As this method is only used inside this module it does not make sense for me to pass ".source"' as parameter.
".source" can be a const value that should be available for all methods, like a classes variable.

The rest of the code is very good!